content/authorization.md

```diff
@@ -1054,15 +1054,30 @@ preceding documentation.
   such devices to participate in the authorization
   ecosystem.
 
+- What happens when a user revokes my application's
+  access to their data?
+
```

Comment:
Shall we mention that a refresh token can be revoked at anytime and that it's an expected behavior and the app needs to handle the scenario properly?

Reply:
Maybe, but there's some risk of simply restating the OAuth RFCs (which I try to avoid when possible).